### PR TITLE
Pre submission refactoring 01

### PR DESCRIPTION
**Pull request by feeds**

### CODE REVIEW

File: GANA-Tests.package/GanaBrowserTest.class/instance/testCorrectHashAbbreviation.st [hunk 1]
```diff
@@ -8,18 +8,13 @@ testCorrectHashAbbreviation
...
 	 version2 := MockVersion newWith: '2' From: {version1} at: TimeStamp now id:stringFail.
-	 squitRepository addHistorian: (MockHistorian newNamed:'MockMaster' startingAt: version2).
-	 workingCopy := MockWorkingCopy new 
-		 name: 'fake';
-		 repository: squitRepository.
-	 self subject model workingCopy: workingCopy.
-	 self subject model refresh.
+	 workingCopy := self newWorkingCopyWithMasterAt: version2.
+	 self setWorkingCopy: workingCopy.
 	 output := (Array streamContents: [:stream|
```
Comment (Blubl) [inline GANA-Tests.package/GanaBrowserTest.class/instance/testCorrectHashAbbreviation.st]: maybe call output everywhere uiCommitList?

File: GANA-UI.package/GanaBranchGraph.class/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+A GanaBranchGraph is an algorithm, that computes  a Morph Visualisation of an ordered commitList.
+The graph is planar and ordered onto a grid of rows and columns.
+Per column there is only going to exist one node.
```
Comment (Blubl) [inline GANA-UI.package/GanaBranchGraph.class/README.md]: don't you mean per row ther is only existing one node?

File: GANA-Tests.package/GanaBrowserTest.class/instance/testCorrectHashAbbreviation.st (hunk 1 above)
Comment (feeds) [reply GANA-Tests.package/GanaBrowserTest.class/instance/testCorrectHashAbbreviation.st]: maybe collective code ownership?

File: GANA-UI.package/GanaBranchGraph.class/README.md (hunk 2 above)
Comment (feeds) [reply GANA-UI.package/GanaBranchGraph.class/README.md]: done

**Review by Lariiii — APPROVED** [review]

**Pull request merged**